src/protocol/prf_sharding/mod.rs

```diff
@@ -742,20 +759,30 @@ pub mod tests {
         }
     }
 
-    fn test_output(
+    fn test_output<BK, TV>(
         attributed_breakdown_key: u128,
         capped_attributed_trigger_value: u128,
-    ) -> PreAggregationTestOutput {
+    ) -> PreAggregationTestOutput
+    where
+        BK: GaloisField,
+        TV: GaloisField,
+    {
         PreAggregationTestOutput {
-            attributed_breakdown_key,
-            capped_attributed_trigger_value,
+            attributed_breakdown_key: get_bits::<Gf2>(
+                attributed_breakdown_key.try_into().unwrap(),
+                BK::BITS,
+            ),
+            capped_attributed_trigger_value: get_bits::<Gf2>(
+                capped_attributed_trigger_value.try_into().unwrap(),
+                TV::BITS,
+            ),
```

Comment:
I don't understand why this was necessary. I'd rather not make this change if we can help it.

Reply:
Changed this because do_aggregation takes breakdown keys and trigger values in BitDecomposed format. For that we need to know BK and TV to get those number of bits. Earlier we were just reconstructing u128 from bits which didnt need this transformation